DOCUMENTATION.md

```diff
@@ -18,19 +19,19 @@ Geocint consists of 3 different parts:
 - [geocint-runner](https://github.com/konturio/geocint-runner) - a core part of the pipeline, includes utilities and initial Makefile
 - [geocint-openstreetmap](https://github.com/konturio/geocint-openstreetmap) - a chain of targets for downloading, updating and uploading
 to database OpenStreetMap planet dump
-- [geocint-private] any repository that contains your additional functionality
+- [geocint-custom] any repository that contains your additional functionality
 
-During the installation process, you should clone all these repositories to ~/.
+During the installation process, you should clone all these repositories to working directory.
 
-During the installation of the geocint pipeline the next folders will be created in your home directory (~/):
+During the installation of the geocint pipeline the next folder will be created in your working directory:
 
 - [public_html] - any public html that you want to share
-- [domlogs] - access and errors logs for files from the public_html folder
 
 Then start_geocint.sh will copy files from all these repositories to ~/geocint folder. Geocint folder will be the working folder for the geocint pipeline
```

Comment:
highlight start_geocint.sh  as file, ~ is not home dir, it's working dir

Reply:
fixed